docs/framework/react/api/router/RouterOptionsType.md

```diff
@@ -286,3 +286,9 @@ const router = createRouter({
 - Optional
 - Defaults to `never`
 - Configures how trailing slashes are treated. `'always'` will add a trailing slash if not present, `'never'` will remove the trailing slash if present and `'preserve'` will not modify the trailing slash.
+
+### `pathParamsAllowedCharacters` property
+
+- Type: `Array<';' | ':' | '@' | '&' | '=' | '+' | '$' | ','>`
```

Comment:
where does this list come from?

Reply:
> where does this list come from?

These are characters encoded by `encodeURIComponent` but not by `encodeURI` minus `/`, `?`, and `#` since those would be problematic as a path param.